Makefile

```diff
@@ -2,24 +2,17 @@
 
 all:
 	@echo ""
-	@echo "  install: install dep (assumes go is installed)"
-	@echo "  deps: grab dependencies using dep"
+	@echo "  deps: grab dependencies using go modules"
 	@echo "  test: test prebid-server (via validate.sh)"
 	@echo "  build: build prebid-server"
 	@echo "  image: build docker image"
 	@echo ""
 
-.PHONY: install deps test build image
-
-# install dep https://golang.github.io/dep/ (assumes go is already installed)
-install:
-	export DEP_RELEASE_TAG=v0.4.1
-	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
 # deps will clean out the vendor directory and use dep for a fresh install
```

Comment:
please update comment to reference go modules

Reply:
Oups, Sure !